tests/examples/test_scripts.py

```diff
@@ -16,6 +16,7 @@
 from unittest import mock
 
 import pytest
+import torch.cuda
```

Comment:
This is cool vs importing whole `torch`. :)

Just a small question, should this be wrapped in a macro if GPU is available?

Reply:
I think the import must have been auto-done by pycharm haha. Personally I would have just done `import torch`. I'll update it!